src/components/global/search/search.hbs

```diff
@@ -1,5 +1,8 @@
-<div class="search">
-	<h2 class="visuallyhidden">Suche</h2>
+<div class="search js-search">
+	<h2 class="visuallyhidden">Search</h2>
+	<button class="search--toggle">
+    <span class="visuallyhidden">Search</span>
```

Comment:
Better call it `Show / hide search`.

Reply:
By the way, closing doesn't work yet. Is this easy to implement (by clicking on the icon again, or by pressing Esc)? Otherwise, naming the button "Open search" only makes more sense.